Lib/typing.py

```diff
@@ -441,8 +443,32 @@ def NoReturn(self, parameters):
       def stop() -> NoReturn:
           raise Exception('no way')
 
-    This type is invalid in other positions, e.g., ``List[NoReturn]``
-    will fail in static type checkers.
+    Static type checkers treat this as a general bottom type,
+    a type with no members. The typing.Never type provides a
+    more explicit name for that concept.
+
+    """
+    raise TypeError(f"{self} is not subscriptable")
+
+@_SpecialForm
+def Never(self, parameters):
```

Comment:
Maybe it is a good idea to somehow alias `NoReturn`? 🤔

Reply:
I'd rather not alias. I covered this in my PR description.

Reply:
Can you please add a comment explaining the motivation? Otherwise next year someone in pursuit of a quick reduction in lines of code will merge the two. :-)

Reply:
Done